packages/models-library/src/models_library/projects.py

```diff
@@ -24,6 +25,9 @@
     none_to_empty_str_pre_validator,
 )
 
+_Unset: Any = Undefined
```

Comment:
I don't understand why all these aliases: `_Unset = Undefined = PydanticUnset` can't we just stick to the original one?

Reply:
Or at least let's define them all in one place and then have clear instructions on how to use them.